### DISCUSSION TITLE
Weather this project can be redeveloped for my own server?

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Blessing-Studio/WonderLab.Override#122] [问题&回答] [answered]
Whether this project can be used to develop initiators specific to the specified Minecraft server?
If I can, I'd like to get an authorization. For server `mc521.cc`

**Comment by JWJUN233233** [comment, marked as answer]
Surely you can
你当然可以

This project use AGPL. If you do not publish your application into internet, you needn't to open source.
这个项目使用AGPL。如果你不发布你的程序到网络上，你就不需要开源

**Reply by MoYuan-CN** [reply]
All right, thanks for answer
